### PR TITLE
Solve Aggregate-Intellect#38 none command and Aggregate-Intellect#23 error handeling

### PR DESCRIPTION
**Pull request by YujingYang666777**
@20001LastOrder @saminegash I created a new pull request from a new branch. The main branch consists of several unrelated commits. Thanks for the review process

### CODE REVIEW

File: apps/slackbot/task_agent.py [hunk 1]
```diff
@@ -105,19 +105,40 @@ def run(self, task: str) -> str:
...
+                return f"OpenAI API failed authentication or incorrect token: {e}"
+            except openai.error.Timeout as e:
+                return f"OpenAI API Timeout error: {e}"
+            except openai.error.ServiceUnavailableError as e:
+                return f"OpenAI API Service unavailable: {e}"
+            except openai.error.InvalidRequestError as e:
+                return f"OpenAI API invalid request error: {e}"
+                
+
             print("reply:", assistant_reply)
```
Comment (saminegash) [inline apps/slackbot/task_agent.py:142]: remove this print

File: apps/slackbot/task_agent.py [hunk 2]
```diff
@@ -105,19 +105,40 @@ def run(self, task: str) -> str:
...
             # return assistant_reply
```
Comment (saminegash) [inline apps/slackbot/task_agent.py:143]: remove comments that are not necessary

**Review by saminegash — CHANGES_REQUESTED** [review]
once addressed we can merge this

File: apps/slackbot/task_agent.py (hunk 2 above)
Comment (YujingYang666777) [reply apps/slackbot/task_agent.py:157]: addressed

**Review by 20001LastOrder — APPROVED** [review]

**Comment by 20001LastOrder** [comment]
@saminegash Can you take a look and approve this?

**Comment by YujingYang666777** [comment]
@saminegash I think the requested changes are addressed. Could you please take a look?

**Review by 20001LastOrder — DISMISSED** [review]

File: apps/slackbot/task_agent.py [hunk 3]
```diff
@@ -143,15 +168,17 @@ def run(self, task: str) -> str:
...
+                    return result["thoughts"]["speak"]
                 else:
                     print(result)
                     return result
-                
-            
+                            
             # Get command name and arguments
             action = self.output_parser.parse(assistant_reply)
             print("action:", action)
```
Comment (saminegash) [inline apps/slackbot/task_agent.py:184]: why are printing here? do we need this?

File: apps/slackbot/task_agent.py [hunk 4]
```diff
@@ -143,15 +168,17 @@ def run(self, task: str) -> str:
...
                 except:
                     return assistant_reply
+
                 if 'command' in result and \
                 'args' in result['command'] and \
                 'response' in result['command']['args']:
                     return result["command"]["args"]["response"]
+                elif 'thoughts' in result and 'speak' in result['thoughts']:
+                    return result["thoughts"]["speak"]
                 else:
```
Comment (saminegash) [inline apps/slackbot/task_agent.py:178]: this is required to see the logs on aws fyi!

**Review by saminegash — CHANGES_REQUESTED** [review]

**Comment by YujingYang666777** [comment]
@saminegash I solved the conflict and not it should be ready to merge. Thanks!

**Review by saminegash — APPROVED** [review]

**Pull request merged**